.buildconfig.yml

```diff
@@ -224,6 +224,10 @@ projects:
     path: components/ui/colors
     description: 'The standard set of Photon colors.'
     publish: true
+  ui-components:
+    path: components/ui/components
```

Comment:
nit: Let's not use `component(s)` in the name of a component. If we say "ui components" then we usually mean everything that is `ui-*`. If there's also a component that has that exact name then things start to get confusing. :)

Some other options may be: `theme` or `styles` (in case you want to only drop styles here) or `widgets` if this is for ui widgets (The name `widget` may be misleading to. But it is consistent with the [Android package naming](https://developer.android.com/reference/android/widget/package-summary?hl=en)).